### PR TITLE
UX improvements

### PR DESCRIPTION
**Pull request by VirginiaDooley**
Ref https://github.com/DemocracyClub/EveryElection/issues/1576 and https://trello.com/c/lhcjgP0s/3121-review-chriss-ee-design-feedback

Logo and checkbox form fix dependent on https://github.com/DemocracyClub/dc_django_utils/pull/44
Date form dependent on https://github.com/DemocracyClub/design-system/pull/75

- Adds `ds-status` to status messages
- Replaces missing data attribution in footer
- Places election IDs into a table
- Separates election types into separate templates with customised tables
- Makes tables scrollable horizontally, as needed on mobile views
- Bumps `dc_utils` to 2.1.2 and matches current base template structure
- Adds `RadioFormCluster` for Election Organisation Division form

Election ID creator - review table

Screen Shot 2022-11-07 at 8 52 44 AM

Date input form

Screenshot 2022-11-23 at 4 22 38 PM

`RadioFormCluster` for Election Organisation Division form

Screenshot 2022-11-24 at 12 16 51 PM

### CODE REVIEW

**Comment by VirginiaDooley** [comment]
[Review Chris's EE design feedback](https://trello.com/c/lhcjgP0s/3121-review-chriss-ee-design-feedback)

**Comment by Bekabyx** [comment]
I know the requirements haven't actually changed as part of this branch but thought it was worth mentioning this to see if you're experiencing the same as me?

The version of the design system we have pinned (0.2.2) is dependent on `jsmin==3.0.0` but we specify `jsmin==3.0.1` in our production requirements file. This means the pip install fails out, which has the knock-on effect of not installing our github hosted packages. This means I can't actually do `runserver` because the design system dependency hasn't been installed.

If I manually pin `jsmin==3.0.0` in our production requirements file I can move past this issue and run the project, but I get a handful of other compatibility warnings to do with dependency conflicts on my pip install. Happy to dump these if it's helpful.

**Comment by VirginiaDooley** [comment]
> I know the requirements haven't actually changed as part of this branch but thought it was worth mentioning this to see if you're experiencing the same as me?
> 
> The version of the design system we have pinned (0.2.2) is dependent on `jsmin==3.0.0` but we specify `jsmin==3.0.1` in our production requirements file. This means the pip install fails out, which has the knock-on effect of not installing our github hosted packages. This means I can't actually do `runserver` because the design system dependency hasn't been installed.
> 
> If I manually pin `jsmin==3.0.0` in our production requirements file I can move past this issue and run the project, but I get a handful of other compatibility warnings to do with dependency conflicts on my pip install. Happy to dump these if it's helpful.

Thanks @Bekabyx This was a good reminder that I needed to update the base template and a few requirements following all the recent changes to `dc_utils`. Hopefully sorted with the most recent commits.

**Review by Bekabyx — APPROVED** [review]

**Comment by symroe** [comment]
I'm having a hard time reviewing this because I'm so in to refactoring my local EE with the image builder stuff that I can't switch to this :(

I think the PR needs some cleaning up in terms of the commits (adding and removing the date picker for example), but it looks broadly good. I might have to revisit after a deploy though.

One thing I've not been able to check: can you wrap the radio buttons on `id_creator/election_organisation_division/` in a `ds-cluster` please? To make them look more like the current site:

image

File: every_election/apps/elections/templates/id_creator/election_subtype_group.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+                            <td>{{ election.division }}</td>
+                            <td><code>{{ election.election_id }}</code></td>
+                        </tr>
+                    {% endif %}
+                {% endfor %}  
+            </table>
+        </div>
+        {% endif %}
+        {% endfor %}  
+        {% endfor %}
```
Comment (GeoWill) [inline every_election/apps/elections/templates/id_creator/election_subtype_group.html:26]: Nit: file needs a blank line at the end. 
Also:
  -  `every_election/apps/elections/templates/id_creator/elections_group.html`
  - `every_election/apps/elections/templates/id_creator/local_elections.html`

**Review by GeoWill — COMMENTED** [review]
Looks good to me going through the id creator, and looking over the other pages. Couple nits/thoughts:

There's a small inline EOF nitpick. 

Testing the ds-status tag on the 'staging site' banner made me think it would be better at the top of page. 

Current:
Screenshot from 2022-11-28 15-36-34
Suggested:
Screenshot from 2022-11-28 15-36-14

maybe not for others though? I guess it was changed for a reason.

**Comment by VirginiaDooley** [comment]
> Looks good to me going through the id creator, and looking over the other pages. Couple nits/thoughts:
> 
> There's a small inline EOF nitpick.
> 
> Testing the ds-status tag on the 'staging site' banner made me think it would be better at the top of page.
> 
> Current: Screenshot from 2022-11-28 15-36-34 Suggested: Screenshot from 2022-11-28 15-36-14
> 
> maybe not for others though? I guess it was changed for a reason.

I hear you on the status message, but the suggested change conflicts with the design system https://democracyclub.github.io/design-system/layout-demo/

**Pull request merged**